editoast/openapi.yaml

```diff
@@ -6566,11 +6558,9 @@ components:
       - $ref: '#/components/schemas/EditoastSearchApiErrorObjectType'
       - $ref: '#/components/schemas/EditoastSearchApiErrorSearchEngineError'
       - $ref: '#/components/schemas/EditoastSimilarTrainsErrorDatabase'
-      - $ref: '#/components/schemas/EditoastSimilarTrainsErrorInfraNotFound'
       - $ref: '#/components/schemas/EditoastSimilarTrainsErrorInvalidPath'
       - $ref: '#/components/schemas/EditoastSimilarTrainsErrorRollingStockNotFound'
       - $ref: '#/components/schemas/EditoastSimilarTrainsErrorSpeedLimitNotFound'
-      - $ref: '#/components/schemas/EditoastSimilarTrainsErrorTimetableNotFound'
```

Comment:
Since this error is not there anymore, I believe there is some outdated entries in `front/public/locales/en/errors.json` now (and all the other locales too).